burn-book/src/building-blocks/tensor.md

```diff
@@ -37,20 +39,22 @@ a tensor from different inputs.
 
 ```rust, ignore
 
+let device = burn::backend::wgpu::WgpuDevice::default();
```

Comment:
Since the examples below are supposed to be generic, I don't think we need to carry over the device declaration. Let's remove this line (but keep the fixes to the tensor creation methods using `&device`, obviously).